lisa/node.py

```diff
@@ -736,6 +739,26 @@ def _check_password_and_store_prompt(self) -> None:
         ssh_shell = cast(SshShell, self.shell)
         ssh_shell.password_prompts = password_prompts
 
+    def _check_bash_prompt(self) -> None:
+        # Check if there is bash prompt in stdout of command. If yes, the prompt
+        # should be filtered from the stdout. E.g. image yaseensmarket1645449809728
+        # wordpress-red-hat images.
+        if not self.has_checked_bash_prompt:
+            self.has_checked_bash_prompt = True
+            process = self._execute(
+                f"echo {constants.LISA_TEST_FOR_BASH_PROMPT}",
+                shell=True,
+                sudo=True,
+                no_info_log=True,
+            )
+            result = process.wait_result(10)
+            bash_prompt = result.stdout.replace(
+                f"{constants.LISA_TEST_FOR_BASH_PROMPT}", ""
+            )
+            self.log.debug(f"bash prompt: {bash_prompt}")
```

Comment:
The log should be like: "detected bash prompt, it will be removed from every output:".